strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/main/java/org/carlspring/strongbox/cron/jobs/FetchRemoteChangesFeedCronJob.java

```diff
@@ -0,0 +1,52 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import javax.inject.Inject;
+
+import org.carlspring.strongbox.cron.domain.CronTaskConfigurationDto;
+import org.carlspring.strongbox.repository.NpmRepositoryFeatures;
+import org.springframework.core.env.Environment;
+
+/**
+ * @author Sergey Bespalov
+ *
+ */
+public class FetchRemoteChangesFeedCronJob
```

Comment:
Since this class is specific for NPM, I believe it should be called `NpmFetchRemoteChangesFeedCronJob`, or at least be in an `npm` sub-package.

Reply:
yeah, but currently we use "horizontal" naming strategy instead of "vertical", so for example we put all services in every module into `org.carlspring.strongbox.services` package, same with cron jobs.
I would like to have "vertical" package naming as you suggest, so every module have own package, like `org.carlspring.strongbox.npm` or `org.carlspring.strongbox.web`, but I think we should do it everywhere in this case and it will be massive refactoring.

Reply:
Okay, I think you should at least rename the class, so that it's clear that it's for NPM.

Reply:
done